content/03.methods.md

```diff
@@ -19,4 +19,13 @@ talk about the discriminator model and how it works
 talk about calibrating deep learning models with temperature smoothing
 
 ## Experimental Design
-talk about sampling experiment
+Being able to re-use label functions would significantly speed up the annotation process.
```

Comment:
"re-use label functions" in what way? I believe you mean reuse label functions for different edge types?